google/ggrc-core
# GGRC-8288 Update Webpack to the latest version

# Issue description

We are using "webpack": "^4.20.2", which uses internally UglifyJsPlugin to minimise the source code for production. Actually, UglifyJsPlugin was deprecated and Terser should be used (according to the official Webpack's doc https://webpack.js.org/configuration/optimization/).
Impact:
1) UglifyJsPlugin works with ES5 standart of js. Cos of this, we need to use Babel to parse some features to the old syntax. If we update Webpack's version, there'll no need to do such transformation. This leads to performance improvement.
2) The latest version of Webpack uses Terser internally and able to minimise ES6+ syntax.

# Steps to test the changes

1. Run the app via `build_assets` and `watch_assets` script. 
2. Verify that the app works properly.

# Solution description

Updated webpack version. Removed babel plugins that are not required because of using Terser instead of UglifyJs. 

# Sanity checklist




# PR Review checklist
